content/learn/publication-opportunities.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: Pre- & Post-Publication Opportunities
+weight: 6400
+type: essay
+abstract:  "Funding, conferences, awards, and more for your Quire project"
+---
+
+## PRE-PUBLICATION
+
+#### Funding
+*Apply for grants, awards, and other funding before getting started*
+
+* [ARIAH Digital Development Award for Art History](http://www.ariah.info/ARIAHAWARDS.html)  
+
+* [College Art Association (CAA) Publishing Grants](https://www.collegeart.org/programs/publishing-grants)  
+
+* [Institute of Museum and Library Services](https://www.imls.gov/grants/available/digital-humanities-advancement-grants)
+
+* [Knight Foundation Arts](https://knightfoundation.org/apply/)
+
+* [Kress Foundation Digital Art History Grant](https://www.kressfoundation.org/Programs/Grants/Digital-Art-History)
+
+* [Mellon Foundation](https://mellon.org/grants/grantmaking-policies-and-guidelines/grant-proposal-guidelines/)
+
+* [National Endowment for the Humanities (NEH)](https://www.neh.gov/divisions/odh/resources-for-applicants-to-the-NEH-office-of-digital-humanities)
+
+## POST-PUBLICATION
+
+#### Showcase
+*Share your finished project with other professionals and the public*
+
+* [Quire Community Showcase](https://docs.google.com/forms/d/1R3mOLgsJCw9vx7PQJlVy8w1TRwgxFMWUiOo8TtuSI_A/edit)
+
+* [Museum Publishing Digital Interest Group (MuPuDig)](https://docs.google.com/forms/d/e/1FAIpQLSeKK2mwSG9fpMgAFoE5qkjKvYUQK7U4596QikNxBB3mE9-ODQ/viewform)
+
+<br/>
+
+#### Conferences
+*Speak about your Quire project at conferences*
```

Comment:
Let's find a different way of phasing this. "Speak about" feels awkward. Maybe "present your Quire project at conference"? Open to suggestions.